vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
+
+# export KETTLE_HOME=$HOME ## kettle.properties file is expected in .kettle subdirectory of KETTLE_HOME
```

Comment:
Not all study data is uploaded using the "make" approach available in transmart-data.
In this approach, the different pieces of the study data set is loaded from a central place based on the type of the data (clinical, acgh, ..).
Some studies are loaded using a different approach where the different directories containing the different pieces of the study data set are traversed and loading is done from within that directory.
In this latter approach, the KETTLE_HOME is not managed by (the makefile in) transmart-data.